src/Fantomas.Core.Tests/Stroustrup/SynTypeDefnSimpleReprRecordTests.fs

```diff
@@ -82,3 +82,106 @@ type V =
     }
     member this.Coordinate = (this.X, this.Y, this.Z)
 """
+
+[<Test>]
+let ``record definition with private accessibility modifier, 2481`` () =
+    formatSourceString
+        false
+        """
+type Person = private {
+    FirstName: string
+    LastName: string
+}
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+type Person = private {
+    FirstName: string
+    LastName: string
+}
+"""
+
+[<Test>]
+let ``record definition with internal accessibility modifier, 2481`` () =
+    formatSourceString
+        false
+        """
+type Person = internal {
+    FirstName: string
+    LastName: string
+}
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+type Person = internal {
+    FirstName: string
+    LastName: string
+}
+"""
+
+[<Test>]
+let ``record definition without accessibility modifier, 2481`` () =
+    formatSourceString
+        false
+        """
+type Person = {
+    FirstName: string
+    LastName: string
+}
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+type Person = { FirstName: string; LastName: string }
+"""
+
+[<Test>]
+let ``record definition with accessibility modifier with added whitespace, 2481`` () =
```

Comment:
Please remove this test, it doesn't prove anything more than the previous tests.